cmds/webboot/webboot.go

```diff
@@ -0,0 +1,190 @@
+package main
+
+import (
+	"fmt"
+	"io"
+	"log"
+	"net/http"
+	"net/url"
+	"os"
+	"path/filepath"
+	"regexp"
+
+	ui "github.com/gizak/termui/v3"
+	"github.com/u-root/webboot/pkg/menu"
+)
+
+const (
+	downloadByLinkLabel = "Download by link"
+	tcURL               = "http://tinycorelinux.net/10.x/x86_64/release/TinyCorePure64-10.1.iso"
+	wbtcpURL            = "https://github.com/u-root/webboot-distro/raw/master/iso/tinycore/10.x/x86_64/release/TinyCorePure64.iso"
+)
+
+// bookmark record the list of iso webboot allow user to download
+var bookmark = []*BookMarkISO{
+	&BookMarkISO{
+		isDefault: false,
+		url:       tcURL,
+		label:     "Download Tinycore v10.1",
+		isoName:   "TinyCorePure64-10.1.iso",
+	},
+	&BookMarkISO{
+		isDefault: false,
+		url:       wbtcpURL,
+		label:     "Webboot Tinycorepure",
+		isoName:   "TinyCorePure64.iso",
+	},
+}
+
+// BookMarkISO contains information of the iso user want to download
+// expected to implement Entry interface
+type BookMarkISO struct {
+	isDefault bool
+	url       string
+	label     string
+	isoName   string
+}
+
+// Label is the string this iso displays in the menu page.
```

Comment:
@rminnich having comments for each of these functions feels a little cluttered since they aren't adding any meaning, will the linter complain if we remove them?

Reply:
Yes, golint complained